app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvUploadForm
+  include ActiveModel::Model
+
+  attr_reader :keyword_ids
+
+  validates :file, presence: true, csv_keyword_file: true
+
+  def save(file)
+    @file = file
+    return false unless valid?
+
+    begin
+      @keyword_ids = Keyword.create(parsed_keywords).pluck('id')
+      true
+    rescue ActiveRecord::ValueTooLong
```

Comment:
btw should we move this to inside the `validator` like `keywords.any? { |keyword| keyword.length > 255 } { I18n.t('invalid_keyword_length') } so all validation will be just in that validator

Reply:
Fixed in 0c4e8d7914b6fc71ddb701a3d3003cc1c1730124